docker-compose.yml

```diff
@@ -58,9 +58,8 @@ services:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: postgres
       POSTGRES_DB: firezone_dev
-    # when you want to connect db with a graphic tool, uncomment ports
-    # ports:
-    #   - 5432:5432
+    ports:
```

Comment:
Helpful for debugging

Reply:
Yup. I thought some of you might have non-docker instances running on the machine. Thought to avoid conflicts.